tests/gocase/unit/type/hash/hash_test.go

```diff
@@ -936,3 +944,235 @@ func TestHashWithAsyncIODisabled(t *testing.T) {
 		require.Len(t, rdb.HVals(ctx, testKey).Val(), 50)
 	})
 }
+
+func TestExpireFieldOnExistingHash(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{
+		"hash-field-expiration": "no",
+	})
+	defer srv.Close()
+
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+
+	ctx := context.Background()
```

Comment:
Just one test case is not enough.

For all existing test function e.g.
```
func TestXXX() { ... }
```
we can change them to:
```
func TestXXXImpl(field_expiration_enabled) {
  startServer({"hash-field-expiration": field_expiration_enabled})
  ... // original code
}

func TestXXX() {
  TestXXXImple(true)
  TestXXXImpl(false)
}
```